examples/LayoutSpecExamples/Sample/LayoutExampleNodes.h

```diff
@@ -0,0 +1,38 @@
+//
+//  LayoutExampleNodes.h
+//  Sample
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+@interface LayoutExampleNode : ASDisplayNode
++ (NSString *)title;
++ (NSString *)descriptionTitle;
+@end
+
+@interface HeaderWithRightAndLeftItems : LayoutExampleNode
+@property (nonatomic, strong) ASTextNode *usernameNode;
+@property (nonatomic, strong) ASTextNode *postLocationNode;
+@property (nonatomic, strong) ASTextNode *postTimeNode;
```

Comment:
We should move all of them out of the header into the class extension. So direct assigning values to the outside the node is not possible. Change of the state of this node should only happen by providing a new model.

Same for all other nodes in this file.